wiki/Tournaments/de.md

```diff
@@ -1,8 +1,3 @@
----
-outdated_since: 51dee80f97242f58303d6569311a1e93da10380e
```

Comment:
Is it really okay to remove the outdated tag? When looking at the diff on the [status page](clayton.cc/osu-wiki-status/de), the article is missing the recent event *The Round Table* and also a line under *4 Digit Catch World Cup*.

Reply:
If I missed those I will add em in my next commit, since last commit is 2 days ago, and new Roundtable update is like yesterday.

Reply:
I think that should cover it, thanks for mentioning it out 👍